Examples/Google/SpeechToText/Makefile

```diff
@@ -0,0 +1,36 @@
+.PHONY : log protos generate clean help
+
+log:help
```

Comment:
super-nit: add a space after the colon? (And maybe remove the space before the colon in the `.PHONY` line?)